Bluepill-cli/Bluepill-cli/Bluepill/Bluepill.m

```diff
@@ -40,6 +40,7 @@ @interface Bluepill()<BPTestBundleConnectionDelegate>
 @property (nonatomic, assign) BOOL exitLoop;
 @property (nonatomic, assign) NSInteger failureTolerance;
 @property (nonatomic, assign) NSInteger retries;
+@property (nonatomic, assign) BOOL mayReuseSim;
```

Comment:
I can't remember if I left a comment for this before.. but I am not a fan of *may* or *possible* for variable name.